ci-operator/step-registry/jboss-fuse/run-tests/jboss-fuse-run-tests-commands.sh

```diff
@@ -0,0 +1,209 @@
+        - name: mvn-settings
+          configMap:
+            name: mvn-settings
+            items:
+            - key: settings.xml
+              path: custom.settings.xml
+        - name: test-properties
+          configMap:
+            name: test-properties
+            items:
+            - key: test.properties
+              path: test.properties
+        - name: xpaas-qe-volume
+          persistentVolumeClaim:
+            claimName: persistent-xpaas-qe
+      containers:
+        - name: xpaas-qe
+          ports:
+            - containerPort: 22
+              protocol: TCP
+            - containerPort: 80
+              protocol: TCP
+            - containerPort: 443
+              protocol: TCP
+          env:
+          - name: MVN_ARGS
+            value: -Dlpinterop
+          - name: MVN_SETTINGS_PATH
+            value: /tmp/custom.settings.xml
+          - name: MVN_PROFILES
+            value: test-fuse,openshift4-current
+          - name: TEST_EXPR
+            value: ${interopTestList[$i]}
+          - name: OPENSHIFT_NAMESPACE
+            value: ${namespace}
+          - name: NAMESPACE
+            value: ${namespace}
+          - name: NAMESPACE_PREFIX
+            value: ${namespace}
+          - name: SUREFIRE_REPORTS_FOLDER
+            value: /deployments/xpaas-qe/test-fuse/target/surefire-reports
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: mvn-settings
+              mountPath: /tmp/
+            - name: test-properties
+              mountPath: /mnt/
+            - name: xpaas-qe-volume
+              mountPath: /tmp/xtf-oc-cache/
+              subPath: xtf-oc-cache
+            - name: xpaas-qe-volume
+              mountPath: /tmp/surefire-reports/
+              subPath: surefire-reports
+            - name: xpaas-qe-volume
+              mountPath: /tmp/log/
+              subPath: log
+            - name: xpaas-qe-volume
+              mountPath: /deployments/.m2/
+              subPath: maven-repo
+EOF
+
+sleep 30
+done
+}
+
+function check_tests() {
+  maxretries=15
+	mapfile -t podlist < <(oc get pods -n jboss-fuse-interop -l application=xpaas-qe --no-headers=true | awk '{print $1}')
+	echo "LIST ----"
+	echo "${podlist[@]}"
+    try=0
+	  while (( ${#podlist[@]} )); do
+	    echo "Number running pods still: ${#podlist[@]}"
+	    for i in "${!podlist[@]}"; do
+	       echo "Evaluating ${podlist[$i]} ..."
+	       podlog=$(while read line; do echo $line; done <<< "$(oc logs --tail=50 ${podlist[$i]} -n jboss-fuse-interop)")
+	       if [[ "$podlog" == *"BUILD SUCCESS"* ]]; then
+	         echo "pod ${podlist[$i]} terminated"
+           if [[ "$podlog" == *"Failures: 0, Errors: 0"* ]]; then
+             echo "${podlist[$i]} was successful"
+             todelete=$(echo ${podlist[$i]} | awk -F"[-]" '{print $1}')
+             echo "DC TO DELETE: ${todelete}"
+             oc delete dc/${todelete} -n jboss-fuse-interop --grace-period=20
```

Comment:
@smongiar To make sure this command don't return fail status, we can make these changes
(since last run failed while deleting DC [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/38586/rehearse-38586-periodic-ci-jboss-fuse-fuse-xpaas-qe-container-main-jboss-fuse-ocp4.14-lp-interop-jboss-fuse-interop-aws/1701175290224447488))
```suggestion
             oc delete dc/${todelete} -n jboss-fuse-interop --grace-period=20 || true
```